### ISSUE TITLE
Checkbox for selecting projects to install to isn't being read by screen reader

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#9578]
~Related (sorting column headers): https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1240642~

[From V[S Feedback (847922)](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/847922)]

Basically the Solution PMUI checkboxes are not being read by Narrator/NVDA screen readers, and require accessing the inner Checkbox controls manually. Probably should announce state on selecting a row in this grid.

- AI reports no errors here. I have to drill-in to the checkbox in order to hear its checked state. I have not been able to find guidance on whether this is acceptable behavior. 

- I would expect to hear "Checkbox unchecked/checked" when selecting the grid row. We have the same behavior on the Updates Tab package checkboxes.

![image](https://user-images.githubusercontent.com/49205731/82275681-5bd04d00-9951-11ea-9ef6-9d63cbeab921.png)

**Comment by rrelyea** [comment]
Given that the item is a listboxitem, the checkbox provides a multi selection capability. 
I believe selection, not checked is the concept to be communicated.

**Comment by donnie-msft** [comment]
@rrelyea  Try the Breakpoints panel in VS. You hear the selected item, and then you hear "checkbox checked" if it's an enabled breakpoint.

**Comment by donnie-msft** [comment]
Related(ish): https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1245157

**Comment by donnie-msft** [comment]
@jebriede found that we won't be changing this grid control now, so won't be able to address the checkboxes. He's going to create a separate issue just for the column sorting bug.

**Comment by jebriede** [comment]
I investigated the possibility of using a TableControl to replace the existing WPF GridView being used for the project list. The TableControl will not solve the issue with the checkboxes and requires a non-trivial amount of work for its implementation. Along with it comes a lot of rich features, many of which we do not need for the project list. It seems it is not the best fit for this control and will also not solve the checkboxes issue.
